### DISCUSSION TITLE
Does the Wheel Of Fortune pay out only for manual and not automatic spins?

### DISCUSSION THREAD

**Discussion by oatybiscuit** [BottlecapDave/HomeAssistant-OctopusEnergy#667] [Q&A] [answered]
I tested out the Wheel of Fortune spins last month. Count of available spins works perfectly. Service call to action a spins works perfectly. Did all my four spins automatically, works perfectly _but I never saw any rewarded points added to my account_...

So this month, up comes my 4 spins (2 electric, 2 gas). I spin one electric and one gas manually - 'win' 8 points each, and almost immediately both spin awards appear on my account. Total goes up by 16 points, both in Octopus account, and in my integration total.
Then I run the next two spins through the service call. Works, and the available spin count goes down. But no points have been awarded.

Is this just Octopus being mean and not giving me the points if I don't actually spin the wheel by finger in the app?

**Comment by BottlecapDave** [comment, marked as answer]
Hello. Unfortunately as per the [docs](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/services/#spin_wheel_of_fortune), and this [issue](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/598) it was discovered the exposed API for some reason doesn't reward octopoints and instead this is done via a separate API not built for public consumption. Hopefully this will change in the future.
